projects/common/src/lib/login/login.component.ts

```diff
@@ -98,7 +98,7 @@ export class LoginComponent implements OnInit, AfterViewInit, OnDestroy {
 
     oidcLogin(): void {
         this.formStatus$.next(FormStatus.Loading);
-        window.location.href = this.oidcUrl;
+        window.location.href = this.oidcUrl + '&redirect_url=' + window.location.href.replace('/signin', this.loginRedirect());
```

Comment:
Wissen wir immer, dass in der URL schon ein `?` steht?
Ansonsten müsste man ja prüfen, ob es `?redirect_url` ist.

Reply:
Ja, weil die URL vom Backend kommt und da einige andere Parameter bereits gesetzt sind. Es gibt auch einen Test dafür: https://github.com/geo-engine/geoengine/blob/22fe2f6bb0bf4c2cdb7318dfa78883ba07829786/services/src/users/oidc.rs#L859